02/app.js

```diff
@@ -3,21 +3,27 @@ document.addEventListener('DOMContentLoaded', init);
 function init() {
     const clickEl = document.querySelector('.error--click');
     const enterEl = document.querySelector('.error--enter');
+    const alertEl = document.querySelector('.alert');
 
     setRandomPosition(clickEl);
     setRandomPosition(enterEl);
 
     initEventWithError(clickEl, 'click', new RangeError('Błąd zakresu!'));
     initEventWithError(enterEl, 'mouseenter', new TypeError('Błąd typu!'));
 
+    alertEl.addEventListener('click', closeAlert);
 }
 
 function setRandomPosition(element, error = null) {
-    element.style.top = Math.random() * 600 + 'px';
-    element.style.left = Math.random() * 800 + 'px';
-
-    if(error) {
-        throw error;
+    try {
```

Comment:
`try/catch` powinno być w miejscu wywołania funkcji `setRandomPosition()` inaczej ta konstrukcja to "rozdmuchany" `if`.
Nam zależy, aby błąd przechwycić gdzieś dalej, a nie w samej funkcji.